test/components/componentTests.ts

```diff
@@ -445,6 +445,20 @@ describe("Component behavior", () => {
     svg.remove();
   });
 
+  it("renderTo() only accepts strings, selections containing svgs, and SVG elements", () => {
+    svg.attr("id", "render-to-test");
+    assert.doesNotThrow(() => c.renderTo("#render-to-test"), Error, "accepts strings that identify svgs");
+    assert.doesNotThrow(() => c.renderTo(svg), Error, "accepts selections that contain svgs");
+    assert.doesNotThrow(() => c.renderTo(<any> svg.node()), Error, "accepts svg elements");
```

Comment:
instead of svg.node(). use `document.getElementById("render-to-test")`. This + the suggestion above will allow you to not need the `<any>` cast anymore